### ISSUE TITLE
TLCC channel meta-data parsing is broken

### ISSUE THREAD

**Issue by g2flyer** [hyperledger-labs/fabric-private-chaincode#139]
We currently get tons of  errors in tlccs of the form
```
... tlcc_enclave/enclave/crypto.cpp:102) [Enclave] [31mCrypto: Invalid certificate: unable to get local issuer certificate[0m
... tlcc_enclave/enclave/ledger.cpp:215) [Enclave] [31mLedger: Envelope signature validation failed[0m
```
The root cause seems to be that we are not parsing properly anymore the channel config infos anymore ...
```
tlcc_enclave/enclave/ledger.cpp:74) [Enclave] [33mLedger: Block number: 0[0m
tlcc_enclave/enclave/ledger.cpp:165) [Enclave] [33mLedger: Process Envelope[1/1][0m
tlcc_enclave/enclave/ledger.cpp:187) [Enclave] [33mLedger: 	\-> Verify envelope signature[0m
tlcc_enclave/enclave/ledger.cpp:198) [Enclave] [33mLedger: 		\-> MSPID: SampleOrg[0m
tlcc_enclave/enclave/ledger.cpp:224) [Enclave] [33mLedger: Skip signature validation; genesis block[0m
tlcc_enclave/enclave/ledger.cpp:310) [Enclave] [33mLedger: ### Parse Config ###[0m
tlcc_enclave/enclave/ledger.cpp:315) [Enclave] [33mLedger: ConfigEnv.config.ChannelGroup.Groups:[0m
tlcc_enclave/enclave/ledger.cpp:320) [Enclave] [33mLedger: 	Group: Orderer[0m
tlcc_enclave/enclave/ledger.cpp:341) [Enclave] [33mLedger: 		Org: SampleOrg[0m
tlcc_enclave/enclave/ledger.cpp:353) [Enclave] [33mLedger: 			MSP Config: SampleOrg[0m
tlcc_enclave/enclave/ledger.cpp:355) [Enclave] [33mLedger: 			\-> Root certs: 1[0m
tlcc_enclave/enclave/ledger.cpp:365) [Enclave] [33mLedger: 			\-> Admin certs: 1[0m
tlcc_enclave/enclave/ledger.cpp:347) [Enclave] [31mLedger: Can not decode protobuffer[0m
```
... maybe due to outdated protobuf files?

The good part is that we don't do any error-handling and ignore all errors, i.e., all tests are fine (and from a "SDKIzation milestone" we are robust ); the bad part is of course we do not do proper error handling which is essential here in the long run (for this, though, there is also a separate meta-issue #138).

PS: Complete log file with more context: [peer.err.txt](https://github.com/hyperledger-labs/fabric-private-chaincode/files/3502754/peer.err.txt)

**Comment by mbrandenburger** [comment]
I am wondering if this also happens with Fabric v1.4.1.

**Comment by g2flyer** [comment]
> I am wondering if this also happens with Fabric v1.4.1.

could be, i was wondering that as well.  But didn't have time to test that and in any case this seems to be a bigger work-item and probably better addressed when we do bigger clean-up in tlcc and/or when we move to 2.0.0-alpha/master ...

**Comment by mbrandenburger** [comment]
Fixed with  #147
